php/php.phpunit/src/org/netbeans/modules/php/phpunit/PhpUnitVersion.java

```diff
@@ -40,10 +42,30 @@ public static PhpUnitVersion getDefault() {
         return phpUnitVersions[0];
     }
 
+    public static PhpUnitVersion fromString(@NonNull String version) {
+        Parameters.notNull("version", version); // NOI18N
+        String[] versions = version.split("\\."); // NOI18N
+        try {
+            int majorVersion = Integer.parseInt(versions[0]);
+            if (majorVersion >= 10) {
+                return PHP_UNIT_10;
+            } else if (majorVersion <= 9) {
+                return PHP_UNIT_9;
+            }
```

Comment:
Some logging in `else` would be useful, I think.

Reply:
Makes sense :) I'll add it. Thanks!

Reply:
Fixed: https://github.com/apache/netbeans/compare/bcfb8f7cf2e9aecd60f51ef32aa97f0fabc61cb7..6ac8eac427e4e3e315bf68f10faf6a2cf5d99147

Reply:
Nice, thanks!